src/test/java/com/checkmarx/flow/cucumber/common/utils/AstUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.checkmarx.flow.cucumber.common.utils;
+
+import com.cx.restclient.ast.dto.sast.report.Finding;
+import org.junit.Assert;
+
+import java.util.*;
+
+public class AstUtils {
+
+    public static void validateDescriptions(List<Finding> findings) {
+
+        Map<String, Set<String>> mapDescriptions = new HashMap<>();
+
+        findings.forEach(finding -> {
+            Set<String> listDescriptions = mapDescriptions.get(finding.getQueryID());
+            if(listDescriptions == null){
+                listDescriptions = new HashSet<>();
+            }
```

Comment:
Can be replaced with `Optional.ofNullable( mapDescriptions.get(finding.getQueryID())).orElseGet(HashSet::new)`

Reply:
Hate java8